client/app/components/_languageChooser/languageChooser.html

```diff
@@ -1,7 +1,7 @@
 <md-menu>
   <md-button md-menu-origin ng-click="$mdOpenMenu()">
     <i class="fa fa-globe fa-2x"></i>
-    {{ $ctrl.activeLang }}
+    {{ $ctrl.$translate.use() }}
```

Comment:
`$ctrl.$translate.use()` is being called several times every time something changes. I wasn't aware of that.